### ISSUE TITLE
Styles how to

### ISSUE THREAD

**Issue by timunie** [AvaloniaUI/Avalonia.Samples#10]
- show where to find original one
- show how to override DynamicRes.
- pseudo classes

**Comment by maxkatz6** [comment]
It probably should be done after we have preview/release of ControlThemes.

**Comment by timunie** [comment]
Yes, I think so. The change between 0.10.x and 11.0 seems to be too huge.
